src/bin/rav1e.rs

```diff
@@ -265,6 +265,31 @@ fn init_logger() {
   pretty_env_logger::init();
 }
 
+cfg_if::cfg_if! {
+  if #[cfg(target_os = "windows")] {
+    fn print_rusage() {
+      eprintln!("Windows is not supported.");
```

Comment:
Probably clearer to put "--benchmark is not supported on Windows". This message is a bit vague and might be misinterpreted as "rav1e is not supported on Windows".

Reply:
Sure, I checked how to do that using winapi and it is feasible, maybe later some windows user can give it a try.